generate_ground_truth.sh

```diff
@@ -11,30 +11,24 @@
 #   ./generate_ground_truth.sh sub-03
 #
 
-# The following global variables are retrieved from parameters.sh but could be
-# overwritten here:
-# PATH_QC="~/qc"
-
 # Exit if user presses CTRL+C (Linux) or CMD+C (OSX)
 trap "echo Caught Keyboard Interrupt within script. Exiting now.; exit" INT
 
 # Retrieve input params
 sub=$1
-PATH_PROCESSING=$2
-PATH_QC=$3
+PATH_OUTPUT_wSITE=$2
 
 # Create BIDS architecture
-# mkdir derivatives
-# mkdir derivatives/labels
-# mkdir derivatives/labels/${sub}
-# mkdir derivatives/labels/${sub}/anat
-ofolder="derivatives/labels/${sub}/anat"
+mkdir -p ${PATH_OUTPUT_wSITE}/${sub}/anat
+mkdir -p ${PATH_OUTPUT_wSITE}/derivatives/labels/${sub}/anat
+ofolder_seg="${PATH_OUTPUT_wSITE}/derivatives/labels/${sub}/anat"
+ofolder_reg="${PATH_OUTPUT_wSITE}/${sub}/anat"
```

Comment:
@Nicolas-Pinon why do you need this variable if everything is created locally?

Reply:
nevermind-- i just realized that you cd to the input folder (not to the output folder)